pkg/config/default_freebsd.go

```diff
@@ -26,3 +26,7 @@ func getLibpodTmpDir() string {
 func getDefaultMachineVolumes() []string {
 	return []string{"$HOME:$HOME"}
 }
+
+func getDefaultComposeProviders() []string {
+	return append(defaultUnixComposeProviders, "podman-compose")
```

Comment:
defaultUnixComposeProviders does not seem to be defined on freebsd, that will cause compile issues AFAICT.
default_common.go uses `!freebsd`